samza-core/src/main/java/org/apache/samza/system/inmemory/InMemorySystemFactory.java

```diff
@@ -31,20 +35,28 @@
  * Initial draft of in-memory {@link SystemFactory}. It is test only and not meant for production use right now.
  */
 public class InMemorySystemFactory implements SystemFactory {
-  private static final InMemoryManager MEMORY_MANAGER = new InMemoryManager();
+  private static Map<Integer, InMemoryManager> jobToManager = new HashMap<>();
```

Comment:
* Static final variables should be of camel case notation. 
* Would prefer to rename `jobToManager` to `IN_MEMORY_MANAGERS`.

Reply:
added final